app/models/tv_series.rb

```diff
@@ -1,6 +1,5 @@
 class TVSeries
-  def initialize(show_id, first_air_date, last_air_date, show_name, backdrop_path, poster_path, number_of_episodes, number_of_seasons, overview, seasons, actors)
-
+  def initialize(show_id:, first_air_date:, last_air_date:, show_name:, backdrop_path:, poster_path:, number_of_episodes:, number_of_seasons:, overview:, seasons:, actors:)
```

Comment:
I converted this to keyword arguments because there are so many arguments, it was difficult to keep them straight.